config/application.rb

```diff
@@ -58,6 +58,15 @@ class Application < Rails::Application
     config.eager_load_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('app', 'validators')
 
+    # registry config needed
+    config.autoloader = :classic
+    config.registry_private_key = OpenSSL::PKey::RSA.new(
+      File.read('/etc/rmt/ssl/rmt-server.key')
+      )
+    config.registry_public_key = config.registry_private_key.public_key
```

Comment:
What are we doing with the key?

Reply:
The OAuth token must be signed with a key, for which a certificate is available to the registry server, with the RMT server's FQDN as a SAN.